src/dss_cell_group.hpp

```diff
@@ -56,25 +49,25 @@ class dss_cell_group : public cell_group {
     {}
 
     void advance(time_type tfinal, time_type dt) override {
-        for (auto cell_idx: util::make_span(0, not_emit_it_.size())) {
+        for (auto lid: util::make_span(0, not_emit_it_.size())) {
```

Comment:
I changed the name `cell_idx` because that sounded more like a `gid`, i.e. cell index, as opposed to and index into a cell local set.

I will change it to `i`.